R/extremes-data.R

```diff
@@ -204,8 +204,21 @@ createDataRows <- function(data, param, rowName, isUpchain, includeRelated=TRUE,
             }
           }
           
+          #Build list of duplicates to apply the duplicate notice to
+          filteredRows <- duplicateRows[rev(rownames(duplicateRows)),]
+          applyRowsFiltered <- duplicated(filteredRows[c("date")])
```

Comment:
this is not catching duplicated daily values - we'll need to figure out if that is something that Laura wants.